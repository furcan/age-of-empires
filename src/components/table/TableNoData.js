import { VscFolderOpened } from 'react-icons/vsc';

import './TableNoData.scss';

function TableNoData() {
  return (
    <div className="table-nodata">
      <VscFolderOpened />
      <p>No matching unit found.</p>
    </div>
  );
}

export default TableNoData;
