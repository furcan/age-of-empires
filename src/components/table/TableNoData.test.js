
import renderer from 'react-test-renderer';

import TableNoData from './TableNoData';

it('The TableNoData component has been rendered as expected.', () => {
  const tree = renderer.create(<TableNoData />).toJSON();
  expect(tree).toMatchSnapshot();
});
