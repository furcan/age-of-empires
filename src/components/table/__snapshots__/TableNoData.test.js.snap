// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The TableNoData component has been rendered as expected. 1`] = `
<div
  className="table-nodata"
>
  <svg
    fill="currentColor"
    height="1em"
    stroke="currentColor"
    strokeWidth="0"
    style={
      Object {
        "color": undefined,
      }
    }
    viewBox="0 0 16 16"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      clipPath="url(#clip0)"
    >
      <path
        d="M.5 14h12l.48-.37 2.63-7-.48-.63H13V3.5l-.5-.5H6.71l-.86-.85L5.5 2h-5l-.5.5v11l.5.5zM1 3h4.29l.86.85.35.15H12v2H8.5l-.35.15-.86.85H2.5l-.47.34-1 3.08L1 3zm11.13 10H1.19l1.67-5H7.5l.35-.15.86-.85h5.79l-2.37 6z"
      />
    </g>
    <defs>
      <clipPath
        id="clip0"
      >
        <path
          d="M0 0h16v16H0V0z"
        />
      </clipPath>
    </defs>
  </svg>
  <p>
    No matching unit found.
  </p>
</div>
`;
